code/__DEFINES/research.dm

```diff
@@ -27,6 +26,8 @@
 
 #define RDSCREEN_NOBREAK "<NO_HTML_BREAK>"
 
+///Sanity check defines for when these devices aren't connected or no disk is inserted

```

Comment:
this doesn't work, it's just going to end up in module docs because it's not directly above a define.